docs/toc.md

```diff
@@ -363,6 +357,30 @@
 
 ### [SlidableListItem](archive/SlidableListItem.md)
 
+## Extensions
+
+### [ArrayExtensions](extensions/ArrayExtensions.md)
+
+### [Logical Tree](extensions/LogicalTree.md)
+
+### [Matrix Extensions](extensions/MatrixExtensions.md)
```

Comment:
The Matrix Extensions are still good, eh?

Reply:
Yes, but with the new template we're using we essentially can drop this docs page entirely, since it only contains stuff that's already in the autogenerated page from the .NET API browser website. There's nothing handwritten that's added here.

Reply:
Except our API docs aren't archived, that's a whole other issue.